FrontEnd/src/UserAuthentication/CloudMessaging.jsx

```diff
@@ -0,0 +1,22 @@
+import { getMessaging, getToken, onMessage } from "firebase/messaging";
+
+const messaging = getMessaging(userID);
+
+getToken(messaging, { vapidKey: 'BIk2IwWnyh6QKW0Y6c4ZCNLuNltnCcaGT4hxKNdJl6wgpvQhYfDIkZZ34iU14Ox5zSflGAPL5dmDRbv7GENQrqA'})
```

Comment:
We normally don't commit APIKey in code, any reason for it?

Reply:
This is a deleted file actually but because I backtracked on a commit version that had it, it got included.